### ISSUE TITLE
feat: Reverse conversion

### ISSUE THREAD

**Issue by cromefire** [ledoge/jxr_to_avif#2]
[Windows will get HDR Desktop backgrounds soon](https://www.windowscentral.com/software-apps/windows-11/windows-11-is-getting-true-hdr-desktop-wallpaper-support-and-better-screen-casting), but sadly only JXR, because... Microsoft I guess... so it would be really nice to have support for the reverse conversion as you can already get HDR images using AVIF for example to get pictures from Movies or some cameras (via RAW) and maybe soon from android phones (via Ultra HDR) and being able to set those as desktop backgrounds would be really cool.

**Comment by TheLowerBeat** [comment]
Yes! this would be increadibly useful since neither adobe camera raw or lightroom can export jxr images. This would become an essential tool for the "take a photo->edit it in HDR->set it as a wallpaper" pipeline

**Comment by spider623** [comment]
hello 2025, windows got HDR support!
but it's bugged and only works with avif... the irony is too huge...
